libs/output-mapping/src/TableLoader.php

```diff
@@ -88,6 +90,11 @@ public function uploadTables(
                 );
 
                 $processedConfig = (new BranchResolver($this->clientWrapper))->rewriteBranchSource($processedConfig);
+                if ($configuration->getDataTypeSupport() === 'hints') {
```

Comment:
```suggestion
                if ($configuration->getDataTypeSupport() === OutputMappingSettings::DATA_TYPES_SUPPORT_HINTS) {
```

Reply:
šel by k tomuhle taky dopsat nějaký testík?

Reply:
hmm, vlastně šel... nejdřív jsem ho nepsal protože se metadata ještě neukládají, ale můžu otestovat alespoň to že se ta tabulka založí stringová i když se pošlou typy